IntegrationWithPharmacies/TenderApi/Dto/TenderDto.cs

```diff
@@ -1,18 +1,18 @@
-﻿using System;
-
-namespace HealthClinic.CL.Dtos
-{
-    public class TenderDto
-    {
-        public DateTime ActiveUntil { get; set; }
-        public bool Closed { get; set; }
-
-        public TenderDto() { }
-
-        public TenderDto(DateTime date, bool closed)
-        {
-            ActiveUntil = date;
-            Closed = closed;
-        }
-    }
-}
+﻿using System;
+
+namespace TenderApi.Dto
+{
+    public class TenderDto
+    {
+        public DateTime ActiveUntil { get; set; }
+        public bool Closed { get; set; }
+
+        public TenderDto() { }
+
+        public TenderDto(DateTime date, bool closed)
+        {
+            ActiveUntil = date;
```

Comment:
Change parameter name to match left side for example it should be "ActiveUntil = activeUntil".
You could also rename "ActiveUntil" into "ExpirationDate" if it makes more sense in the given context.